rel-eng/push-packages-to-obs.sh

```diff
@@ -258,6 +258,7 @@ while read PKG_NAME; do
       sed "s/^ARG PRODUCT=.*$/ARG PRODUCT=\"SUSE Manager\"/" -i $SRPM_PKG_DIR/Dockerfile
       sed "s/^ARG URL=.*$/ARG URL=\"https:\/\/www.suse.com\/products\/suse-manager\/\"/" -i $SRPM_PKG_DIR/Dockerfile
       sed "s/^ARG REFERENCE_PREFIX=.*$/ARG REFERENCE_PREFIX=\"registry.suse.com\/suse\/manager\/${VERSION}\"/" -i $SRPM_PKG_DIR/Dockerfile
+      sed "s/%PKG_VERSION%/${VERSION}/g" -i $SRPM_PKG_DIR/Dockerfile
```

Comment:
Here you are always using the version from the `uyuni-base`, for both SUSE Manager and Uyuni.

Don't we need an `else` so `sed "s/%PKG_VERSION%/${VERSION}/g" -i $SRPM_PKG_DIR/Dockerfile` is done with 
`${VERSION}` coming from `web/conf/rhn_web.conf`?